From 632fcf8e59c0f466bc4d62170f765dc9c9bc2ce8 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Mon, 26 Oct 2026 13:17:29 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh gpgkey-E88F5E48.asc from keyservers/WKD: same certificate(s), with self-signatures our copy was missing.

Assisted-by: Claude Opus 5.5
---
 gpgkey-E88F5E48.asc | 396 +++++++++++++++++++++++++++++++++++++++-----
 swaylock.spec       |   4 +-
 2 files changed, 359 insertions(+), 41 deletions(-)

diff --git a/gpgkey-E88F5E48.asc b/gpgkey-E88F5E48.asc
index 80d3640..3af1165 100644
--- a/gpgkey-E88F5E48.asc
+++ b/gpgkey-E88F5E48.asc
@@ -1,6 +1,6 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
 
-mQINBFb+7ZIBEADKchyt5QILvWeM/fi5Jf9SQ+2f4ltU7DVtC9Y2pWrG/Bua6Yoc
+xsFNBFb+7ZIBEADKchyt5QILvWeM/fi5Jf9SQ+2f4ltU7DVtC9Y2pWrG/Bua6Yoc
 wWMEo/XeT7yu/4cMDTH2aRR+9mSir3sVTIjfX+Pnfxdds3/zKZgrMrK0jvBRoWxi
 RTIkFw0L7FqEj0D1LkzXqysLQshRn11AbijoYPaqksr9g6Zk1RgA1p/51mEMLL9v
 AOULKbRoxhTNcfQQwnzhNRygocOJ3QF4ZyVUBkxOfxun7aZQz0OfW8YskincoUGk
@@ -11,42 +11,360 @@ tY2C2VGJDo3albvlOHFJbCT5AVLPnX3wl+5Q2tV6Jd+1806iW1Cl/RQ1DYUzVAQ6
 atFCOl0auNQpCjvFay6k775IxjUPGjTJ+m24SiQbwwxl7JANrgiPSRdqdY7r2IaF
 O1B0gZ+qW9+yb7LzfU1gvhibrIXSsipHDwZE31bv2v8xaLe3PgjBNv5cNb2bHE0s
 V1LQKU0R4EaaACFXS1zEcZol+sivnanbLCTq5kr1VSR6FFAnDtpQh1v5uQARAQAB
-tB5lbWVyc2lvbiA8Y29udGFjdEBlbWVyc2lvbi5mcj6JAlUEEwEIAD8CGwMGCwkI
-BwMCBhUIAgkKCwQWAgMBAh4BAheAFiEENP+VJs/vDpejQOLkD9574OiPXkgFAmBJ
-CC8FCQsrTh0ACgkQD9574OiPXkgiSw/+J1MBHZNscZEEjxPSKX/PX3vgNicBfIlA
-V5eR/bt7NAmqNj0G0mvW++rIRXgOcxKOO46L8eX9/9mg90QXI1H8aNh8lez4ThSS
-Fim2VF6bQGI8ky4DsRlzSt3X/jfjR06XLdbSGpvBiUWn9aWSoAeICl8mR8sMxY+V
-HUbKHRGe6dHSA0Y+DRuQr7rd2gOaSVF0n1B/UgteN8tIDxWKwUu721BijDckTvIw
-rQ7Oh3fZcZMCjU1YlVCDd7bUILYNznb13xJ9CMX4xGuf5+QNbHx507idLfCUg7vO
-qkZJtSFDDE+2EBMIHHQGXyPhhk9umy6/5ENJE8WgFXHTMQ+Q2ADx7YCkMPKRgibo
-MtI5vCrR1j0Rlu1WNap0U7jx2qPQ5axE1AmLHJkG8jUo5TQ0uwmGs7CUTSmgMj0d
-8o8CdZ3zrlLZMu55RMkSp1mNrWRrxmeGAc1RoJF9XX5CeMCggYELmvkgLsFykpI0
-+RhiI7916zmGcejahXnG3sY84XO1rDR0Uvot88iSDApwzASJ33ukoS89w65Q2z1m
-x4QahLx9zDLETtP6F5yXYWtvklm682pS9Xz9HDJ5cZ2fAJBdMvMhp+ciTAqQ1bT3
-Nu3jtT6utgdtFGYYChByRz/OVvTM3F6AyW0sLP/YMyqAeRfH61HRyJAR5X1N4M9/
-G1ge6PBNT0y5Ag0EVv7tkgEQAMX9XJNL82oYajxpmQBC4BYZ5L7pQuIs4w9N5m69
-QmflMqlSHFB7aAPtMPJzqquCsvvL4HP1izmmFXMI8EgeC2xVRiyxMA/0z/cWlcRG
-R3h23y3sZ+d6PqcVRLS/mrbnvFbReMvL9097tZkJ0EeEAriyDjwn5AIbF0M5kFtn
-w2yzgKI96eajZQBfKnhpKi+DhEBiEOWxIrpzmvZOQpgsSpHKcAcjsRpsw+O7bm2K
-ACGulwuQgxt01HdyfAw9vxafH4jYXCfigBpJvRCKgWjt8H2ALDypgALV5N+ugYwM
-5UGlEPRYSnrHNL/OPKewrCizkbaNtqGxy1u3j7TZFlFi28/6AigPX+c2ELO/MVm6
-7JRLHwRl932+Ne2ZDCiFd3kDD7CdE1Pajllfg3+7DI1BJQiSoenNgsHOTAks0djX
-SaeSVBD8+8ePPwV8XEr7P5u54X68X5NfftE5KwJb2Xg1/3jNhKd8QpYSE2GOVw+G
-KgbNW1fm+IbJej+kWeBXg1p/DFurGkKlPVV33IgH/bgU5HjPhdb2WxuH3hPKOzRb
-boYZM9dWaxYyhfQhXi+rI3MWG17o3QdX1yODOzDPwOqOt9XBnR3jCV051kXXV4AZ
-vmBopsn4ptRktrwrxcS4H8nV3ZkbLaMmIoeHxxp4LGK9Qv+DnOIas4gfIE2yoYPs
-nM6rABEBAAGJAjwEGAEIACYCGwwWIQQ0/5Umz+8Ol6NA4uQP3nvg6I9eSAUCYEkI
-UQUJCytOPwAKCRAP3nvg6I9eSG2aEADDvbUNRAcRg1s7C84cjnDIGxxHSOuY1OPm
-xiXuVjaqwHCi/Xrns6hAzHV2Lc43aYNSjoRc3O/L9M0sU2YMULT56jrkBurbLwIW
-qOr+VTEuqk0Xqe8F0BJz7fDPYFm10jR/gCWlOxgoNvesVzVklTt1aJF8Rc1KyaJM
-UdY79feJPEZG6v/LYBbIJ0eQpNQV84HLkommdv3WmffLpq0Y+aclepaBdyktx6Nb
-NLbBdxGnnBX6kOdo+mLCo2dl9FtM+RiWyg5CU4vGjfEHPjRY9rbDgAaO4CSFiOwl
-efGrHibl+Qrc1UcXVRHgDQYLKTnOCY4WhGy0nTIaJ7XLqEDsNjIArGo/yUyWJjGk
-tGeMW4tsoiBHLIx+ZnPn/Xzk/YZIOVZkHv8g8NXmBLb5Bzvf3LLWLulr7cuCrkwS
-ni3ezjsuBM1V02jTIZ1JMQhVH+dUPgN5JJdJ0OgoTVQRgtolIMCkDvaqngzQ5D51
-vAFyX/VUcdMZyTBPF81/TFl6V28gDJpWnpCJBbvno7Lvg4r/22S/VZyofn+QBsEo
-x/fIFaq936EMk7YWztGfOr44xHl6y9KCxGKWyI2trZLl9IBF17rY6tw+yqYtiKOw
-cHV5ME0Mez36wGeJpo4IwUt68Da3c+vLHRYZlizJdV1ykaaLvgP9RadLPb9NQGEq
-PHS2cohkvg==
-=6ev5
+zR9TaW1vbiBTZXIgPGNvbnRhY3RAZW1lcnNpb24uZnI+wsGUBBMBCAA+AhsDBQsJ
+CAcCBhUKCQgLAgQWAgMBAh4BAheAFiEENP+VJs/vDpejQOLkD9574OiPXkgFAmnb
+yd8FCRS+D80ACgkQD9574OiPXkjegBAAn5BbBH1Mg7rncAo8txh+W9O7d65zp3/9
+qsbJvI5bOGCUsWDoDiGR7WSIa6yXpvxHHYGXKbhzc/tzMBOaP9d4aE/yh/txoR/+
+2UMJotwp1x1sRsZskbP4SxUWDpWQ9tjhR4cC4AQUQI3370n0YpFSpv/GZuv3XgVq
+SSxifhI5TmpWFPuD5gweFw9EyYbZjmeE/RsBB4/phmXppdyvvdkbq9nPPBklbws7
+zTyg57lgcVbPolFxeO01Run2DUdRgRXvgR32Syp0a9v75LuiI3zcrrCNXgeaS/Pu
+arJGg79scb8hxhp+YrRTKsPUrBDweZjhlVlFIuXe/RvEGwy/hizv6XKlPpWJ7Qgt
+H+EDtxqOhCcguJNJRmv3vN3gkv59opjSha73Bkk7fg57C2dJ18nKaLQ/yXAuP93S
+DPc8iCxHeXxAriHtoOL/Bi8jmWlBYcx8loys8uZBWHTDADWCQNthEQ1EWrpTLrkQ
+tHq6tjjzRU5QUnCZi0pOtOUqV82piO2CZ+GcsTMEW8+3l8qiY8nsVE7OA6FXiXp/
+QyhExWRyqusK1fjqToVFo53SFIx5jWjk4mUO8NHAv6AiWMdQggPiiHH7KSDHsZLl
+OKHp+6ovIEjmm0DhExEL7SXqyDXRgR8P6ISHZ4sTqHvbRrqu6KcolfwWLLGET22p
+VT9ppbB/XobCwZQEEwEIAD4CGwMFCwkIBwIGFQoJCAsCBBYCAwECHgECF4AWIQQ0
+/5Umz+8Ol6NA4uQP3nvg6I9eSAUCZ/poYwUJEtyuUQAKCRAP3nvg6I9eSBBHEACm
+/u+EfydjiOFjBBrPYbaJAWTl2c8KMqlbi/CgQzDUzFSiIyPHrKq0F73WUTzXHh8V
+N83ZaGsb+YylxGEZJYcox6krG3N1Ldxqj4lm+GtS5o7u1+F3hbNNUYjHzF/f4kJN
+eZQIvp8caZw/sqwrBsh079OsV2d/1XTWlrlynbGKfQ1U6UqCYSE8rs34k3QTiFoo
+Eb3tllM2SjXg8cgPldTRPWT/yfWOZW4L6LCASnR6+mz1dTzlQPKMhRq9KvCuQZsh
+5QeW/Tf/EuZlqAjQ/W2Flx75JGq5KdT3JRHUYcr5jW/KKvppf/FLmy55JeQGdY4V
+pOmxnklCo+LOlhqlaP1O4YCURzyy235p+3Q1Sm6fJ3OqanF5hfolFiMBh4zMy2ui
+QWUrmOewaGN2yuf752gLzLz59vD/RFOfLkhXsLas1eatK6Vxknk4oXfd9+Ol1afm
+unZ3KXQw9PA1CwBdZR3Dk6QQseyTl4pRAQcSBIlYvW5sXTWJdjET0f75iO22UPt4
+csda1/K0Z02lSG34Ev9UVSnTw84u+XflqO1xWk/vWzQJyXI0MMdqlgXoujXVuTrS
+5pEWj+sjwwexhuv4UeT1EKbeWjP9jX7IF0coixE10ROEH+rAs0m/LLGDYgFyMznt
+th64JR4XPwGAs5aT/k99mer0/osfkMSPmWN7q+R7WsLBlAQTAQgAPgIbAwULCQgH
+AgYVCgkICwIEFgIDAQIeAQIXgBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15IBQJmDUWK
+BQkQ74t4AAoJEA/ee+Doj15I88QQAJEsRwFSKhwTF3gFuyv7IFqYCf4T34tlv3ZM
+yxX6LyT+uU/Ao0+Jqsgdm3Ge2ijU5KZUVXfpXRQAN8i28Qu1Ko3XbXUie03fwecU
+neUMfy0dww1BIMjZxV1DUlrd9z+X39qoMxfhQpXLXc2bzAGK1immlq8gRIka9k6s
+QbDaSccu5brHyY60pa/p/ikoFTOJcXbw5b4HW9jrD5Z4vOA/Op0mQmTojZqVObpo
+qRuoSQ9ZI2DzQQ5YcIo66kYIEIjTFK9r9MdYV+mS10CzEuP89QNB4gzUDYFn5SIF
+f7Ne+N2nkdZRtRqZSLRCQGNIotsAJAefE8989Rtc5+6WWtF8Ud0dlhGEZNq43A38
+HLfkhhP1I0yA9/RmpX7NBzxQrEw62MnTrssH4Z5ZH4RBMD/yr0v/kBwNsFvRde7W
+z8gzQ1xfzFxA7zPFvQBp6pLMgPTGTOpE1d5nyoCiMByrgNn9jujCN90b3k14RDfL
+nhpAQj4jfylSJDKmOtyPv5ZkgCI1f0N2ccn3Dno4rNnMShRsxbHiACOnRs3Tsd0m
+iJlAtypQXWPvirD0ciWP0B4kkGAnpbFKup6H9i9fUvjVbMSC27yvgEOL7MUHa2Vu
++3GJmMDFcdcYoV3mD+9sRBRJrCAxIrFYdA5dP4C4iAiBtbjdIfx39mTM6MC3IyhU
+nwSR/k7CwsGUBBMBCAA+AhsDBQsJCAcCBhUKCQgLAgQWAgMBAh4BAheAFiEENP+V
+Js/vDpejQOLkD9574OiPXkgFAmRCMC4FCQ8kdhwACgkQD9574OiPXkgiURAAyNmy
+8tRmUMBwTkh90819OQIq/EWgrKj0f+3OnP2nk/tnC+dQX6MCAuc485im2qAOXQc5
+lP5evDvHUjpbnumfAYu+NqeG3rA92PCY75IJD/8ofbmZ707+omKuXgUXIzJ17Jmz
+pI8bTV8bCCPfMAfgHJOh2u1Ecbu60sg6iuV2eUqPhWyUyNuwfUaXrFt17V6reItj
+QUV3th7Si7vng7pylm2qe2nB2CXQtgfRhKg+oqx/bb2vIuB5Ufi3x2ERpnoVDjhb
+jUVpjmN/ehbpi0m8ZkqHnnxbYachi/zNo7H9ICJIaKHpBCiGIzqfrtm0KORODfvw
+fDUjWi1OZ2U4k2fBGdE1TC7QzwOzVxNt9Zf7VXvYg/SXV02WIe6BreTMDozojq6S
+mIaPomyB5IlrmS9s7sOxJuq2kpTMIhDAQ/MCeq79zRJH40Ql5DHZ26rfKXtT4ejw
+tL19uNNTqBkNbudaNcrmk55PeabAg1WdXZBjlIHUWKWLt1T6Pw7L/KFSffr+T7od
+kRzb+NN8UcyZNVq0Ar9JPj0ieqoEIMCdZ5n3WDEahIpHIEgjpgENot67rM2vdhiT
+v3NtYuSFO0QFgEf+YqxvL3inra/hCjBTCeEgxW+MtyDkEaz/aJngsctqJBXG6cIa
+WQV/lwHs6svL+tBd/RH74t8fE3hTHKHDFQoykGvCwZQEEwEIAD4WIQQ0/5Umz+8O
+l6NA4uQP3nvg6I9eSAUCZCgsrQIbAwUJDwpt6AULCQgHAgYVCgkICwIEFgIDAQIe
+AQIXgAAKCRAP3nvg6I9eSMtdD/9uaAhZlFxl3AluKU7i+RXKdf496U0bL0TBbuHT
+E1te/cbljrlcWbJzhRnyg/2Id/VUK8MHJphqmN/jRqBixiSgot2vqOREI0ZdvkPK
+CkSGBkoxoYGXuAy+Vgrdso39sQlv1rYTs4tNK13ey7wL4pPDjKJWzLSMU2lbFnPD
+GNZRa2dhCTNsD8CoZC4AQc0Nrnyd1eyJxh1hHGVI65xioYCJGUtmBEaqhCZ4JaW6
+FVZuPJCEIcIFNYZ+ypzM0f2QUJeXF6KqPBviZ9zel9xe4R/S3Gwa9M8xWMSmHYzr
+8TV8xH09NMbsgb80g67DNVbQPeW/rL9Txb6JCMiGO3wfnaM1mYvo5guik4QaVifl
+VEtkRD7ftNf4G/dNY56Ix5dRbIcifmKJ3DkptFxmpVtSIPo6BXvgeUy34/rUuRMF
+g4QU++jnohGpMn99UqzGVa6M9yrJdRljHz9vOagdvGe8Ima4bn57wVSxn2mYw16l
+1a+bvlzgEYHrFxws5bsJGbcEN4lP4Tculxg02lw8PhuOQdF760F74gIIOkb0XjXS
+OhjO41qpsU76+Zn2lBWieTloJ5eho5+de9t/DQ/l1nETKSVvr9g0kXx1QGH6N0bg
+cuvJi4b78yyBc79+jlArggEvgUUnNUHJsJrlbQaDd/Lqi5/ofcwwBNyt6eDnMBrK
+Hi9xB80eZW1lcnNpb24gPGNvbnRhY3RAZW1lcnNpb24uZnI+wsGVBBMBCAA/AhsD
+BgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15I
+BQJkQjAyBQkPJHYcAAoJEA/ee+Doj15IhkQQAJCkuo15tE6yZHWKqElbUjCatUlQ
+dvCq78dkcvs8eeDQl2dpUYnMc4i8qIdYJSpPDjcl92c5cuFqul6Symtk77vhNRPg
+Xi9JC2vb6kLVHAuRsDfl96XhKtKWOqBs4T7rea8Hb3ClqjXp77issxw62QYWn2+g
+Emux/a+AMM+SKFi/1t/fVDNqUez3OAOjaht064guYm/Ycz5BoiNUsbDFYALqRCEw
+waNG+nNjP+pG6b0oi3MSBQOpbRher5/Hwo5kO6QJJWvLguaD44iuTfqo0oT+xx3h
+zKXHtTGMQKzcvW1hXI/Blb18ZSIcCMjBotxrDi8k0NZtovERLAdN7ybR8S0c+Jk3
+f5UFk5qX24NzZJgZNBHeuh1TBHlQQCc8fi1Zbv/0epJL7xlrfo2EW/B6ywITJjYR
+QGYACnn/cSM4h9rzphk5CkchxyDGOAmr51r12nbodI6hJ/hyvfDJP59OPjmlUO3t
+uJfuxsFX7G6v5OJQdiN1rzOsKDhQsBXPYlUXip0T60tmuJHWjjAgnnWImSlxCcpx
+V6evFeDVorW/1zPeRx6g5ub8xjCa6KZyCOtcsWJbxtg8zLDanrVWTfy0K4ln9hHJ
+ubX0o5+0AlbnZDEGQJYq/yTHk2ANBGadh9myZ2RnoySs2DwOqzXIk6W6uc0Wpsh5
+B2TTzuWoCiggx8khwsGVBBMBCAA/AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIX
+gBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15IBQJkKCf6BQkPCm3oAAoJEA/ee+Doj15I
+FhsP/2vPJvSCBZJ7jVwxlx9PS2jvntjhUdDZfwJ/AgHEst9SRp9xWzoIIoDgo4Xy
+1mKXQjmciHWu74dFTkW57iNLWv//blrhMB6oT/iFer/jSZwl0wcsOslIWaZ3sr8l
+yS/b6LVcuxqSML9cZBMXI5bPSot/4PGkXFlaz3f1QxNMJ/RFpfwyPOOfgP0npBCl
+pb2BlQqYuVu6WlezewFkoo1LF1QXXfUtIW73Rw5QzBrWakkLgOOpR4YkPFjREtA1
+4NK8jdDiOzhJngJ9ZpZutT5QUAWUcwjzbpNXaD7k1ZlQSClkoJEFnnO8sPMcD+Vt
+XP/OGOuEdcj7rSmHqy2z9ocdzSgtJshX5D9SrYfel1gH55xEsRXB4fpandNiBkwY
+Jxi8g2t0pqRJYJj/64Vigt0cXvTjhsbUnS2gDXB6sfnX0/XRPNNIa+eu7Ke2AN24
+6kgNPCp2UiKZCF6OXP8Q84LzL0OMcSOE2n9yY6q4/ri19jN8RIelSPW/HSP0jnp6
+/iDd/abEi9FYFgSNTn8M9NZU34lksUfHpGw37HQi6kyBGY5MM60bViMxh/wMdULr
+OgUN9TQ8VZd6mMLuP+jIxeek6Humri0fcn3krN7LnZ/QBimnV3PBTD5FYQYy7muo
+WWFRj3fI5TV5btyr4yxskQl1MZdShbTerwIPPVVLSkl9RB4mwsGVBBMBCAA/AhsD
+BgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15I
+BQJiRCI5BQkNJmgnAAoJEA/ee+Doj15ISFAP/ihufU+Wgb3Kavh9lov7lvOxJFmF
+thv2u0yJZbHeCQ0k5yMt00XkNCAGF4MxKCtI7l3HyZKVOdx2s833kRcUTew0HWeX
+zeiHjPdBUVrU3QQ6Smrqu76eHhgT8GUBqM4iJ1gB2yoSne1149UCbrpSJ+QYyXRS
+Nj046cMBVjKmzhc/CNEaIWbhI5E9RqQ3l+IOmPpTLZs231t3QQUZBraZ0OZpbMs1
+IWnZaoEYmSjw2pdcHjdKRTDx1u2BesDVWEFsmOWnlXcZYDI6gjiGnj7kYqNqq6d6
+2RWWeeUZPEwgHjWccxxcCUcN3ntGQhFMGqSGUdipenA1yeB61Qscnx05c+vwGM3R
+xj3+yYL5OkH9F0MuOgj/UmHpJxEsoJMGOFrlzpf81Nrjuxo1EhG+qVxy014hMBeJ
+I0CCb/cV6a86VEVHb29WhLNMN1ziOttlXcgJrvYYTvWIrfbN6Yh+tHBG386fbYVy
+iwwRoTHpJGK4SNFrVNN4RH1AqMJQfAaS+8GUl62XHPZVDhj4LL6f3WydMazHroUt
+dIHs4VO4xboGE81eZmuOLK3TR+45olqpU6vz2lHkn5Nuwu21H0j34wqB6v44G0Ob
+UtacZG+H1+QnhAH9OiVYkzuuCOT6qg1agNOtZiN5+Lvr3T1KWe946cWqYnTZ+ZWY
+SnlpQI0j6F1uR+ISwsGVBBMBCAA/AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIX
+gBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15IBQJgSQgvBQkLK04dAAoJEA/ee+Doj15I
+IksP/idTAR2TbHGRBI8T0il/z1974DYnAXyJQFeXkf27ezQJqjY9BtJr1vvqyEV4
+DnMSjjuOi/Hl/f/ZoPdEFyNR/GjYfJXs+E4UkhYptlRem0BiPJMuA7EZc0rd1/43
+40dOly3W0hqbwYlFp/WlkqAHiApfJkfLDMWPlR1Gyh0RnunR0gNGPg0bkK+63doD
+mklRdJ9Qf1ILXjfLSA8VisFLu9tQYow3JE7yMK0Ozod32XGTAo1NWJVQg3e21CC2
+Dc529d8SfQjF+MRrn+fkDWx8edO4nS3wlIO7zqpGSbUhQwxPthATCBx0Bl8j4YZP
+bpsuv+RDSRPFoBVx0zEPkNgA8e2ApDDykYIm6DLSObwq0dY9EZbtVjWqdFO48dqj
+0OWsRNQJixyZBvI1KOU0NLsJhrOwlE0poDI9HfKPAnWd865S2TLueUTJEqdZja1k
+a8ZnhgHNUaCRfV1+QnjAoIGBC5r5IC7BcpKSNPkYYiO/des5hnHo2oV5xt7GPOFz
+taw0dFL6LfPIkgwKcMwEid97pKEvPcOuUNs9ZseEGoS8fcwyxE7T+hecl2Frb5JZ
+uvNqUvV8/RwyeXGdnwCQXTLzIafnIkwKkNW09zbt47U+rrYHbRRmGAoQckc/zlb0
+zNxegMltLCz/2DMqgHkXx+tR0ciQEeV9TeDPfxtYHujwTU9MwsGVBBMBCAA/AhsD
+BgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15I
+BQJeX30GBQkJQcL0AAoJEA/ee+Doj15IQfUQAKJXvpXgq4cbxiiReLc0I5pFDl2h
+eGSMfubNNjS6QNyM6C67AXSLI+zyQ4nr2Z1KO/6pkzNXwXgxI2AzzCmZEc9i9RxN
+lJlPfoCvxicXEF0Z0RUdXa+MV0USp8t+bhLFF0Brrw+pefcN8plheYGlcy7nCAvk
+0QkAAeUrwOBoSWqHHbCWxFUmNiOOL/6xVEiWlH8ezE+VxAqG6xVWN/ycEMCTiSeH
+Kln729GVc94uybjApS5lrlZsUs6Oew2vKe8AG7LG5Es3mzRKDkd+sHw9iZBvivMj
+T45F7rLy0YCKKLB1gzbghQqS3FJW1tirMHOW5Zim6tJsT2FbdoMAVjB1cqicoCAV
+bNF02q2Ivxaf9cpMovRJRCwIOgbFRX0GNkKO5+LwSqOoJXKV0ibQa+FvDon92OzM
+tLpE4mB3Rh3IHCQlHDiEp7eBXYBKpUMsKDEdDcRltwARDboPx9I6bf48Nk/MsvkI
+wWv2DNLrfMOjfiirXQ+GlHcbNKTvVXALOLQTXSPPpYs/L/N54CtbeqkQKD9tb1cM
+N/vGHSwn/RKFeffXfuQQpYQrITV9QadPbPN7u4CPUjGP6R8ekZyTwhk//UlgdiD1
+M1HDWfgai8Q3h8csuLSNQxNY7KG9lU/y7RccJ8dYhBUZrUNWTHTllG/XbV0fORUK
+rt5oOs/saybUCCt5wsGVBBMBAgA/AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIX
+gBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15IBQJcq5X/BQkHjdvtAAoJEA/ee+Doj15I
+17MP/17f6BJtjiKT89Ii5XtvpfxKXQ1gxlL+fTW+Ce5QRTKh7c7OMqZegfoHvsum
+YreG90/zQn3aPs4pwp8heNk0SiXQyQF7XcA3OBgVu0xVa8tGvW43uRDsKEkAojuV
+PCwdCqxsp2HTsEoEZQoYmIh0anZAJkXRnlMeB2JiDbJJ8gQ19ix2Ax2RrUSSijZc
+1hJbXAn6yFaV1D+GjzQlyEB2ekCMieeORwLfhNwFM/j/HUPUOE1Vjd+rXRpLbpO0
+QO3lNSOsSRXLEuHHUZnFB0Vzk5MrCOAAIhLdHRGrC6NarYuLtp66ppEGz2hW0q5A
+FkH8aAOm+6xSBk/YqCOiszxWbHobqvP+Xmj6LPDHY0/vArPThE8HLntOzgETuhXB
+2w5SbUw1BxCGonsEXGEoVrM9rTCt9arLafFpuA8tJ/LECY3FN0mpjbD5kJVilR+v
+ogxGrGO/o4iInq5lrG9z+BVjmDy9KQHpbBYzZTVSD6OGRlBHafauQ/BLz2uiFE0D
+Ptl/YzgaRkak+XOC+9Xpw6EBmw6t8+qEmEbR8XVYSse1yuEEWG62PLTpMjOAfv4x
+NElO08soWpxtu1lb6J8cyVeKW9spX4tajyOQ4Crd6SNu4T+VK0Wxd1w5ECLFe0F0
+179vn8Zj3/dHUumb/0Jq3x4XjIOkuZ/KQ7vZMHCm/PchyY/dwsGVBBMBAgA/AhsD
+BgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15I
+BQJcha+9BQkHZ/WrAAoJEA/ee+Doj15InbUQAI7Z5/0i9wseeUOEw/WXVb5A6Tvt
+pcIEdwKRsfq2FI+qnjeVe9SHz5lFs+zWiISuj1uUTDoTfGka2WGLLyo4OT8Y8muy
+umaLxN0r2/XL6COc9ACq2qST83sugrBJG/0rKYYlpShvoiIo6g3Kk5zqbcJ0oyhw
+mG/FtuG43In5apGiDT4cUqMTr97Tvqvaf6kTprpgxvFKuXrViFB/7V++XloBS8jZ
+RH/V7Kgje4grtENbnlvcV5nEljpu7V68dFwUXwFCRItHrjYtrSihXm/pak/knl5l
+t4dWzsT5720w7FgyzknbDbg8oj++jctMtPho5CpZ74nVb37K/C1KDHfvAM4v1yVZ
+NWfBBU7LBbbqV3wJxqvr/tQRSTa66BQT7ViIjFRsvJB77WZ22lGRmzFupKSAMwhV
+xx9MXBiDFLZwRnJ7x2YtPmaTVJd3xfrO2OWpFlzQ3oZHlZQLvQ2+GDeCSLmySCFO
++ZNvJCKwwEhwCzmPNOlWD0URQljEvQM73sy3WDWKCA98H2bHB+daGriemKrhc6C+
+eh7k/Fo3utXXe0xFvGghbsiFTmUkFCk3VB9rtM/9wQpAcgTg6Mw6f88dTvuxPVbH
+/Kk8azxQxnNc9fhDeJWgNs41eFvYhYf38iDurt7pg+niQCCX1bVyao10TfIQYiae
+SRgJAMGVaEGGwUetwsGVBBMBAgA/AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIX
+gBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15IBQJawkRKBQkFpIo4AAoJEA/ee+Doj15I
+eFoP/AmwUfqQNDLMJSyS5PfzNOO1XQ4K7hnkK8xLVdyM5SKJvcCLB9VgnxOcE+Ip
+Ndp7HAO7SAk/7aOQvqbgjhENP4sjqfgmIiXcrmNUnHC/+ZfiveVQRiWYqt+lqp12
+BS0O8JQhdRtz8rpsfDCR5clf2GtSsUZTpavmX/1iSFKkTaXb3+toFwSYoT0BJx1M
+Y1GargDiKKGlXGTO6+4kJCl+akGcNMG5kqgV1ZI19DMzPg1UzVgtn8PngBtLVbdx
+pYCZkXPlwx6paRSMqHJ0mWs5pt+VcYCdUE3Am9EyW05vYpnP0T4Dbyzy2z+b2S/t
+We7P2faoZQ4yK4eGjd/7mMnaqs0Hb/l2JCxE3IXjU9rJEXpvRURmYd3bc8eNeMrV
+2Tb8JoHUFaJYidlrPXtDrdAPGBfh4RxhwDS8l0p/AwbfXIyYQ+QBjYEdl3vTootu
+iIb1P0ZIkLywUUdlKT/jNGSI91YL5ASSl3MXp9ss7DA99AzBS5L0Fg2xd0NxdzHm
+FYHUJLvJMW1zDbrJMvF+GhxD/ntARlDWQQO8q18b01qFqI44mn+LpVHx15q4dh01
+Ok2bSucDfU5vbNtPFesBPD1rJJMVBvDBpS6oMhbmCedHq9mT4wr2Q82eP8b+z6Go
+7Xc8ZRUuYtHOk1qfI8FvRMSw9dACVuRWtlnlkUbvGrkIYC/jwsGVBBMBAgA/AhsD
+BgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15I
+BQJY4CZcBQkDwmxKAAoJEA/ee+Doj15IyfUQAKhK3Cfmky9+hr0KL1I/eqXGpT6w
+EAjJ/xGphfL1AADxMdWfN0zkmCUq3quKAr7ZmgNCsYQcH1fmhN5T4v0uEcQho3rv
+cGYhLiBInr6xq4ZkaFcRPN+gm+VdNZ+cZ+18cale7jH3yDutLjyjs4bgHTsbytt8
+88Pe5Zy+c+2fFGVJkng37Tgwh0yT1AkCbhMaEqdCaMFWPo8zwGkhXIN61dNlED9a
+toSB56XJt5AjmWlyWROIMt7kcUPxaFaXMymgBgXgCG1hiQ+ZQ0kCI24eIotamr7A
+4yXYesB4LnA/bCMNJTHZqD6cl9wsavD7JN5xstS0t6QxQnzk8dVGpjxxckSdGHFy
+8cGaiEi9WrgmKW0HwgYi7SB/GjnGuWdnLWa/0IVhdMDYtOiKo90ynmSsq1yT7mui
+u3uhN/1Hp4dD8duVig769lOW6h/EXSJCc9jORXHasqZydkDzhYXIth1xr3kzZDcr
+ihcBCZqXyylXnnzJKlWbYUq0ACxM6QMN4ijIBRq7xtU4E9QSURaiZq+Fq5so75bp
++wRVCgRskAX71TP4z/t8k2X+YWrdDLILKoMdgw1uCcjlAs2TicCgipnelGjHsPX4
+LRaLy0M/j60HaoI0702HSJj+zNs65RAQVJpWjDki+zh6ynZ9H+LX1pWr88K02FwZ
+7DuKHvFjoKeztWlhwsGVBBMBAgAoBQJW/u2SAhsDBQkB4TOABgsJCAcDAgYVCAIJ
+CgsEFgIDAQIeAQIXgAAhCRAP3nvg6I9eSBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15I
+cxUQALUOgEk8BptqCRdiJJ6zgFTr5riBOYkOPxq+poR+NhqbzdQ0pAOREpln31eY
+lFai+AA99stwc0czAbdQzpIaTU4MTYnrRgcVeiNsBcq2UuOPPb1HRKen08A84zqp
+e1MX6xDi9tUjwnQ8XEFNNSZAFFzSCYT9Eu6N24ECdlb0e7T1nDVnik6tZ1rT8ct7
+zGZh2aq0h4nsGsCUJtWhBt319QEUbyKPCufTzah3wHURdW1WPzP7HDktzBif2PaQ
+TXuD6v1AYePU3kEHgfBliN9UzNAznGjZYNzix/7KQtdwjtzahr/aKi7JlkImloNG
+QJz8Xh8/ox3OM4pahYYIgWx3pxi5i0d+fTZAmPADFR8DXgiJtKO01vsSWWuVxFU9
+gTm/vx5KN1w+g1PIwxJUWMbOJ0Hsvi67jnzHoYWDMGD11GSiCf/OhYY91ArEHbPw
+K6RjP5aveq2kZfm2PFWEPevqSvuao2JzxVml6zeaeFpAnQ1fP2G+C8BRuKNSVbAn
+E9NJu2mOuVpbWgZJCBg1gzlY14Ce4/4vYBSysedyGL/PbWjfxRJsaJWp7g4gS3+v
+jzc4VhS3RPAvxBCuR24Z1Coca75DinVnN1XUNOV7Tu0ahjLCwmXqURv2hHEl35Zx
+bIWyHr/QdCOg1RMhkSNztIFkFe6Q1yOlrLEvMrMZtCbhMQAPwsFcBBIBCAAGBQJX
+F9IUAAoJEJ5pae6SCFcEpMAQAKoq5ZGZ2XDVxcdhINyKvKCNfiZASaOYUAfCCFSM
+S+88diQCcsEwChmvyk0gqQ+c+SW9jHZrxRceIxQjGHmCFS8NkigqPgryDgQAdhsv
+LiXNn11Q4IYFygCgyhu0kgAiTt+ng9ofkjodpskZy0aQ22KU+Tk0uPAh9pDjLCL4
+6+9rrR3EXixGWUbzh+Bls/IskWsx3gOVAyV4Lp9ahQSeOkc2mJq86cOWJK4gMKZj
+lsHGfK1GruShW8fJ8OzxPwY2GiVSE69yMvChOnze/58iX9nYSpXxX8k4EhQNErMe
+o3ffzeHiFsvFo6MNFIYF2PiKoCVKTRhItadQqAY9MN0kcA56Vj43HkmU56nQhzFD
+RgQjDAnRwYrdC+ZNlGgEQqqVOLnpO6sZIvMqbGunM6gI42kXHdUVnzgBtLU0OG25
+5Nmv4iG+et4gZwEsWdbjv3/6h4fzeo/8cj/A4Y/y9fHt2wRT59zgxRI+8FmCdWlf
+4KclRamidCTrp2ZMGk0oqcBcSOyNvUBYv6lIQFXm/msVVWH/Ry7YIG6csEsg83LV
+SpsCgEjVn/gdQSOQGcnA0VBy3v94Du8Mv6c4fOuehCIYKu7iRO6r9nNTOuV4+My3
+HALL9E/QrN1LCsBaNa5k1MnuIOhLyY/+nTnZkQNFrRJV+5bY2fwTFMaXyHUbf9de
+g196wsFcBBABCgAGBQJXDtB1AAoJEJOAjRaupNGcLucQAKuVGlIzdVObnt3cEIf0
+M4OFbcnuRQk6Fjo7DY96B/gwxM/CG4HedtEdXUYCZrKY8lFPfAOBfU/wcAJRWTK9
+3WXlD2ByHU0RRlzeB0+EGYoogJLf0Vp8dJS7gkRmc+vt7UYgwB3dczI6mxAL4Jjy
+tAD4cYRx9L0wlfTy3xHoxZdgcsOw2oo++TbJLppvYPEIXupqrrk/MO3RsLqwZYg0
+C2rx4B5w14Xy+fZm3B7zWI6+5wTf+yTdGZZm92V1uPoqG0+jyjP9GnAVTGwPaL1o
+jG0Z9Db/LkpQjPwf2z0ADRIT1bZZFkkUTYhpZhPeCQBytcZS8uqfTvyG0ibY3SLG
+KlwJuPwFFzdi5tjEYhEjdLbLNxwHsqeuPf8xbbqOrkydKvjc+cRIN0Yv2K/EozEu
+of38E8VQQlNL60D/q6fF+BBmRLrTscbVhVIjLT5ylYZJ8R+K4ORAZPCuGHKXMzF8
+F/3lG4YzyxFmvbOVv3r4bIz6L5cPnjaVlHj8Kb0vmCxBz0nJPrEapjq0kvuL4sMn
+9jkhfeu5oBQEzClGZ7mdZPsNKQzIs7ipamcTWyfx1CjdmObY60kDm7TY60D557GU
+OMyq2IQaaR6A7nQ6ruWF0P28LXnABCvKEli9IQiB29EnIArf09xnEc6ASKTcUFv+
+3htAVywbQ+oK3xWAP81PhvygzsFNBFb+7ZIBEADF/VyTS/NqGGo8aZkAQuAWGeS+
+6ULiLOMPTeZuvUJn5TKpUhxQe2gD7TDyc6qrgrL7y+Bz9Ys5phVzCPBIHgtsVUYs
+sTAP9M/3FpXERkd4dt8t7Gfnej6nFUS0v5q257xW0XjLy/dPe7WZCdBHhAK4sg48
+J+QCGxdDOZBbZ8Nss4CiPenmo2UAXyp4aSovg4RAYhDlsSK6c5r2TkKYLEqRynAH
+I7EabMPju25tigAhrpcLkIMbdNR3cnwMPb8Wnx+I2Fwn4oAaSb0QioFo7fB9gCw8
+qYAC1eTfroGMDOVBpRD0WEp6xzS/zjynsKwos5G2jbahsctbt4+02RZRYtvP+gIo
+D1/nNhCzvzFZuuyUSx8EZfd9vjXtmQwohXd5Aw+wnRNT2o5ZX4N/uwyNQSUIkqHp
+zYLBzkwJLNHY10mnklQQ/PvHjz8FfFxK+z+bueF+vF+TX37ROSsCW9l4Nf94zYSn
+fEKWEhNhjlcPhioGzVtX5viGyXo/pFngV4NafwxbqxpCpT1Vd9yIB/24FOR4z4XW
+9lsbh94Tyjs0W26GGTPXVmsWMoX0IV4vqyNzFhte6N0HV9cjgzswz8DqjrfVwZ0d
+4wldOdZF11eAGb5gaKbJ+KbUZLa8K8XEuB/J1d2ZGy2jJiKHh8caeCxivUL/g5zi
+GrOIHyBNsqGD7JzOqwARAQABwsF8BBgBCAAmAhsMFiEENP+VJs/vDpejQOLkD957
+4OiPXkgFAmnbycwFCRS+D7oACgkQD9574OiPXkixlg/9HldpDzsGlx972cfCNaDa
+WWxCvepAKFYtMi9EkIoA1qJ/rgn6FcBSVFSgB1Svuqa+3zmvXM1v5OLc72l1WMjm
+M30sdciq32QQV4beLoASfEJSDNc6/dGCTP+CyHFN9SohP3zc6Uhxt7ltRp1/tiXl
+oki5bHYKXl2nAZmvU+G6rkkphXHYD6/0uSvKr0TXMsBlqF9B5QlCmKliVVCypzaC
+riHludyR43r0VL18ZIWmqCbj0AThwtXJ9pJvfIXTFEEtjV7jX5ehkS7H/o2+4ASF
+GYXhUKV1TbN0GAVj8knetvALAbt/Cmy5Ad55hiDTi8rRSTwJRFM/M7558av89b2r
+Ssyw5+Oxy+CNLEDu6xzl9F11Qc6Fsepv++tCTtoHmUa2kL96qMaAzcmbfKXNU9Ko
+aH5PTTuhQY6L2jQOcpKiKY8JOFo8DcY0RerCbMsoQO995hTX6T7NVTLek/EF+n79
+H7P3b1KAk1ti/q9eYoH7dfSBld7xu9GGyaFMqyLGGdr6d/gbh/ovk6faFNcKIMe3
+Ok1qAgm6eqoW/b6PrS71HpDJu6Gcg6O/0jfZ+eY+SisqXuaq03Ktk3RGI607deoX
+hJFYLsL8p4a8m4r8/Ok0w0YHf0WLgb35RfNdqZ7VY2tqpM2P/J+tRVXdgpOFD9bO
+mhCw6cGBlaJPP1Lz7HUPdDrCwXwEGAEIACYCGwwWIQQ0/5Umz+8Ol6NA4uQP3nvg
+6I9eSAUCZ/poSAUJEtyuNgAKCRAP3nvg6I9eSAcrD/4lY/JrXOImEk5d/BWUwchg
+Wt774rBfs10s5NgQqs/qz6pr9o1NO5VoKSTbMBqqZ+2AaPfUrM6K05iDl+F5H7rI
+eU+8TcjZgQ4Qv3JPpt0M3EJHKElX+UAqKLHPcEahycqLsuGXvJB41LVnsHgu31Ft
+K/eiA6Eg5JSByqv9iJOackIxykauL6UsTSOT5GdZpBhRfkPiWB4K2zUMThK4dnRF
+yaWF2FtpjqnvG1xU6gO3mBl/KZCJi09rubliIh6/5RjwqAzW+gIXtAvYBeAp6UVn
+sueAFO+Xk5BgFlh0JP6UN1LT7OVmGY3ieOGFQ3Kb7aB5ZtXA0Io2s31WOF2lhq7e
+huYJVqQzJ4NaVPsQDiw8xBDR7KthdelD9Rns+DeCIbVijxOiNbPXDpmkFN5hst1x
+Fn/1L/f6habIT7p3uiWIlldj/370v2yw0K+kyRFdgUWTB657AwI6+BEqwpPcV7bz
+lFnSRAdHflvu1i6gI3OtMVVR+/szT1tvLNZfSNc06WuMXE1bkiHqKaN6lzWow8KD
+dHKnpiaL/+yltnU7FfrW/3+8XjB5txNjSifMWXAzHHWy/tnOv6uDJIihKlEvuaXZ
+uLJ/sxo3DoF8vYBRVS2XKcp7m0UYCDlTIR0Cl4hH9JOX9Q0eTF2eB23d9Ua8SBXL
+J00Ip49dtDuyKjlhm+bg+MLBfAQYAQgAJgIbDBYhBDT/lSbP7w6Xo0Di5A/ee+Do
+j15IBQJmDUWbBQkQ74uJAAoJEA/ee+Doj15IX8IP/2hJJWvqKx2y1iZWsereEebA
+Le3MDOVn/K8eFwK0LSfYszYGw8HzCvaCDXXE0RQdQKvPve/ofLCNC0H8L0KRCJO7
+UOkDrVVTyTU62LAw35H82Im/TYF73sP9rLtg4sWdb0mspGTcsnEJzewp8OibV1wi
+l4ue0PR9ARaMmNPSm1M/Q/vR5zJCxTQh3gasd5cQfeREUEdoS+8cQmT8seZcWrkZ
+pwSVGi4YwTX4H9pS5qPpybSeHrMMkiddtxJNdIT2UdckHD5kYN1tjec6AoWZubRw
+OowzbGOanIi6cfG7YmS30eKrkbxE9eUmlM5ZoZIaO5A0oZJLFJn+hv3dIz2lfjc2
+aVBn3L+iZjyA80iopboiqBeqvTIrxsAMWRIezKwHnoTyPgShbAqvkD0s/+W9K/hA
+Y1E3GldX+eZY1YcHTobYkV7x7np5VUGBNhiugd0mQ1xlIwONs6o3rLGvKEBifSUI
+spKAKgxS9E6URJI05X7H6VZUxe2wCYUpBcSizde6fDXzyrMf9YcqS40my7ZdN9ox
+Ul/G8xhr7uDflxKc3uiruZgcFHSGP3qomfFRFEMrGz5rgQS2XKd1g+7TYHQbwCab
+Ssnu73xV0EsNJ/gzX3Q2/hSZhzy5aKCmZG3rBF4xbvN2hh5ko07p5DnmRY36MGHR
+LulDQDuptqvjB0FD10HPwsF8BBgBCAAmAhsMFiEENP+VJs/vDpejQOLkD9574OiP
+XkgFAmQoKAgFCQ8KbfYACgkQD9574OiPXkjFjA//Q9uDRPZJha0+ALZj4fcYrTqv
+dAH6GzLfUEnTswGVKvmSQhNILJiZFqY2Pdj//8uwOnhD7ORr2jOc4NBcGZRivxeC
+pxNY9OVYKtvdMc3RUx+iClmv1eStfXPpuf84NlLdjY4761VnFoqAsmohZWTLT5FU
++mB1Mrw+aHui+IqwjjWv9hnCrgQuQV0vIl32O7mM7FO8mIUWM4Mz4RZLzQXjOaq5
+3oa4kJcKPVFHmzmo1Bllx/FuIn62IVOlLqQn8upzZkknPISqo4c7cRJ5ZeKQehwg
+ECuCRJ6JJmGoPVkXxXptmOvU5FqW0HQcPGu1b2YzRqEm/95HF1uOHP/vC25hFdC2
+STDe8OMx5gQtAqaFBqDySCu40OCRMmLNh+H1a0S/nN9Bi2yqXKeDQeLTJzqgYNq9
+YIfvV7GVd4OQVgldMPMvpScphs/MliZgMNrLgU8VfrsqjH1Mspcg+zg7kV+i8KzH
+usLDIEXhEkDcjyoaWI4paS3kxlglAdWz0t/bkAZ2ii/8Kx4oNRnwUY6/OTV4mE2L
+pWs3aJp3UDUG7Nnqm28DCUWEwQaNFhoSOZgOHnHLijkbW5p7G5fW3K29dd7YOQxU
+aiyIJLvdEojazo8IfftNRHHk6J4wTw11GnxMnSgrnvEK/oGaTkpOXNH6G5y5X2WV
+Dw475OxFpHGoCbMOKwXCwXwEGAEIACYCGwwWIQQ0/5Umz+8Ol6NA4uQP3nvg6I9e
+SAUCYkQiTwUJDSZoPQAKCRAP3nvg6I9eSKVdEACfxvKKGT5q+y1Y5JpP5RDtpTBa
+30fiDODy1jiJvv7I54gVLlrvhue7VUG9Z3e7rw24W2KdMaRIXOJ6oJ6TqEhSia9g
+0BO95uOLuXKWScfG0oy7TF1/eF/lvqKMxBf+vbRRD0jaRAERqyIgbueaOJhNiefT
+pk3e/k6+7pTNC89W0gXhzdUyKC6p6xYmS0eXZ3At026RFaybPpEolhI15qQhrEsj
+aKqN81Kf0ZUuy6ragu2DJhYHWEVImLl4ptmsdJuYPimM75fnqwiLSfxsVFv71Vna
+Sz7gRow3TLOq7TgFLl1nE1Q37UVYWFVCZinTj68SJzJMKrXF2Obmga/hyPM7rzI7
+53N7V0OQj54Q98mXzAKyckRJluCea4PuKRxBggKVigk2tgiZ7DEfdwGU+/kMS7/g
+IR9Wq7ASj6t6TIvbaILS9/NUXT0OqjOzXys5lNIV+Q9MiEmuhV5N7s0JsCqR30uU
+UXf55+z/oE7NnwxXZaT75PaIv4sNipfvdTq6eizKMHxllFviC5b2QzHx16rWTgda
+Bt+c79ExUO1yZCrb9bXELqDVxAxbaCGb0Km578Z+Epbh33aiNaPYAv4xsRo5MIk/
+yvMKZ91AfMk7sFVunCqVfRtNXJRp3yRMmepxEFK0FcHj/eKcIyPEgM+KJeSzcMWO
+4z+Ljlgr6LJT2mtREsLBfAQYAQgAJgIbDBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15I
+BQJgSQhRBQkLK04/AAoJEA/ee+Doj15IbZoQAMO9tQ1EBxGDWzsLzhyOcMgbHEdI
+65jU4+bGJe5WNqrAcKL9euezqEDMdXYtzjdpg1KOhFzc78v0zSxTZgxQtPnqOuQG
+6tsvAhao6v5VMS6qTRep7wXQEnPt8M9gWbXSNH+AJaU7GCg296xXNWSVO3VokXxF
+zUrJokxR1jv194k8Rkbq/8tgFsgnR5Ck1BXzgcuSiaZ2/daZ98umrRj5pyV6loF3
+KS3Ho1s0tsF3EaecFfqQ52j6YsKjZ2X0W0z5GJbKDkJTi8aN8Qc+NFj2tsOABo7g
+JIWI7CV58aseJuX5CtzVRxdVEeANBgspOc4JjhaEbLSdMhontcuoQOw2MgCsaj/J
+TJYmMaS0Z4xbi2yiIEcsjH5mc+f9fOT9hkg5VmQe/yDw1eYEtvkHO9/cstYu6Wvt
+y4KuTBKeLd7OOy4EzVXTaNMhnUkxCFUf51Q+A3kkl0nQ6ChNVBGC2iUgwKQO9qqe
+DNDkPnW8AXJf9VRx0xnJME8XzX9MWXpXbyAMmlaekIkFu+ejsu+Div/bZL9VnKh+
+f5AGwSjH98gVqr3foQyTthbO0Z86vjjEeXrL0oLEYpbIja2tkuX0gEXXutjq3D7K
+pi2Io7BwdXkwTQx7PfrAZ4mmjgjBS3rwNrdz68sdFhmWLMl1XXKRpou+A/1Fp0s9
+v01AYSo8dLZyiGS+wsF8BBgBCAAmAhsMFiEENP+VJs/vDpejQOLkD9574OiPXkgF
+Al5ffVIFCQlBw0AACgkQD9574OiPXkj33hAAvjmxkhe0r8xhpeCshVd7ptYJIly9
+dvY6q7E8hbNm1NfpqTSN+D3TFrb67T6mm36uGRbTjAxIsTTvq8fmhbF2JDMrLMP2
+Ded11hTTE5cy5/OtV5e8Q9DckwiULwQ15O6Hg2Wi0RiwjU5yzgJA80teVufWhnBy
+ndfIXzqVStFuT72LJcx8IsPlRJOc1tuaE4RaS4+bEX4HjIajOBA7dRldSbpoLbug
+tiwvdfA8AsBXTtbaG9sPTMLzldo8p9Srlq5vD1TZIkCeHh7TCKiJmTb9aEbgH4Pf
+75fED/AGYOrHUJ7YVbMSz8Ik5C4fNFrxvEEo9Rlu5dztKbqIsKqXVGTKszj0yzY1
+GmNgLf2Z0tBkPPtvLrgliRa3iy4fOFS9JU7TQBWawPssA0NsaVwvV5tjHvI1bpls
+0BzS+asfsJKmSZ/OnaNDXbVir0EAd9eZQE5IN5/ic3F3tKL4L6l1k/zjR2bhRRaY
+AlfxNV0yR4GOsawUz568OoPYtm1YQtJkv0Hs5dxWoa6iMLHtR6FD4l7cVxSQi7CM
+DFWkCwGpPqMGjDDLo6l8htxx5hopNtY7ppWfAXNsiUN/vbAhYLtG2xBU7z8X/gD+
+ylVaqLfyc4WFgOONI2bLbCJH+SouQJS5zEF9T8cCfN0wuD3fsm+ZqaV3b+QL3V7L
+zBMU/fs2yg13uTbCwXwEGAECACYCGwwWIQQ0/5Umz+8Ol6NA4uQP3nvg6I9eSAUC
+XKuV8QUJB43b3wAKCRAP3nvg6I9eSJGeD/42XEQZnACwn/rxlNBe5d4C5RaT4yv7
+L/PgJ4+zLjle0f/DeQ/lfa3yHpWFfxF+HfTvhUa4Tl4ZYFuTU+iyDXpfFlqfXXvp
+aCDjJ3nf1azil2N+MNOUyxT4f1BNf5khhoLwxFSpuLVaLq0B9NtMPbHXRnEZk8yZ
+VV/O/wzddHPb4jiIozC1Y1qA2g0R9hDVOAQBGkCYM44uBA9/GQP9KMfTLV0dPlT4
+t9pZogCTza4PBLX3p3tlQ1z8dNIg+y4ADcjU8Kml9uzItmU3IPMs+MF1/m95o52h
+SH3NfxPieiqp6eQDrSnnPZGb6eGEI4AXolqFS0EWWbOyfML0ccxAm6DP7ZVpNgVF
+pd9o7Xqy4WufViQUVqbLmXkh6IJQxbYrgpBQk/uR+IHFZzKBr38+e49+YKjS2Ht0
+QouiKALWagwtxPZlINFZTT5Bd5GJyygccxpdLLNLWULJJFxdOT/U+Ln1vhTsTHkv
+WXFLgedWHXr8bej+4hZiPX40e3t3xT9LbyDNZ/ZdxwixKtRakZRRbFKNeg/kvFYY
+jZKPiUpOo6Pyg4zswwYQS3rcVzlG5wx4ekE9xIZO0qfzcXz/3fcBxbHPrnNXQCQp
+cnYYD4O8UvslD7QMftZOyc14XPtKfyoLUxsj+0+nUEHYm6KpE1dYmH4GonAVQjuK
+fNn1nGx1/7h1ccLBfAQYAQIAJgIbDBYhBDT/lSbP7w6Xo0Di5A/ee+Doj15IBQJa
+wkRyBQkFpIpgAAoJEA/ee+Doj15IObIP/1r6NhoOCUigC20Qa8g/ENSahZTfoxme
+h+8GpZROgCSDF5aYyigizl+67ilNKrh6Y+ZgoUicPeXSajEKIQTsmFKQ/N4of1Ar
+mZx8If+F/vZjc2ckUtz/Re1H7wUvZeRUTpGV24fEgtDOlfe8SjeDWbxHvaQH5Ylq
+EzNNvFtI4JpUyZ8z6YqQBT72YcA1NgUbumbTr/GwbXC98z6UP1WRn/+i9+xMhr9t
+9enxWh9Po/eALKYkTjKF/gzkKgQSyhQXgIeC4bBMJ5MygjLXLpu2uljnCzukocxH
+SJG8siRg8iJ4TE/PeObey4mc+sR1sr1L9pbCmpzH1iBZ0YeKJFwkDLM4Q6C+54Jl
+K15lTi2abVznl4oJhcx1KtonoYhBZyFwSuHbyVjnIfWVO5b06cC3xAUAJDt8LRMM
+76CorcBkrWSCT1PoKbbcJ5BNjFgOffe2H4gJylKMFuHq8uICPx54a4gS4KHuAkjN
+IZ9w3pERoM6DGmgS5nw958rUUUt5DZySFtdavx1F9SdCMSTHXfX0Et7d6lTX/+h4
+7kt4ZMMFeQMv4oIFMXlucfHz9YJv+EFqkSp7qh3GXTfTTfCn5Y0vsOVeigF1HH93
+P0qMNm0wd8Cpl6f/9fk/uvqfYXOtNP8T/aan9pSxhvYxHz6yTBnVg871SIZAAS/b
+J6GTqMh1EVJFwsF8BBgBAgAmAhsMFiEENP+VJs/vDpejQOLkD9574OiPXkgFAljg
+JkQFCQPCbDIACgkQD9574OiPXkh06A/5ATUqPCqVBO410r4U0t6MTlLsNcXFyJoa
+OzH7Ma58qsMUHyBuhO0/77ACtkCM8hGN79gg3IWe6Ds7cqGLip/oQqqoV+3oGWZn
+f7go2HekqZ5SrUKIt7A0o7Uv7iyqmA1MfNjVwfiX02dToHdbyM1WQG42o9ZbZsxP
+FmjTlLaHP/OWKY7Bpsb2eJAx4ek9orTs/mSIPb1Zvm6N4yE54EDPH+KaIGZWYzx5
+1uGfiBjt2KbQmH4YTLayPMBjFHGDACsnV0w8zNw1n9RemWBvjid6LDowc3xl7HuG
+DyShVicmyTnP8iKVp2YmQUQQRZpgN6T3lv6fcjCaH8+P8HozZgpN7tljWdKtQxk4
+4QW08rpslPZBQCzlEwSaKWcqRGBozSSmDxJycYusBHjnxDJ0dK5FpBWj/kx/c+fW
+IT5k5MRL6Lx+MxBnlLhz7N02FPxH8vLzoYwMC4U7kht6ARgw6/efbyf7wVxuVLiq
+LJP5YoITHpW9p3b96lz+rskPW15kJNpd3geHuZvwKQVbq7vnCBmZcdEekHnkxrMt
+d+qvzbqIpfAgObT0QBzQlzgnK5NGgZbV5JKbODIkEBtt2NQQGsweJUaz+cYISS17
+DbTzFY1X45mFWcjGtLc1AXc/zvv3eBlL7J405Do9IblRVQS8vsLerU8eI/pomg6S
+kmulFxafZmDCwXwEGAECAA8FAlb+7ZICGwwFCQHhM4AAIQkQD9574OiPXkgWIQQ0
+/5Umz+8Ol6NA4uQP3nvg6I9eSAoeEACqP6EJuPnMnbDPKvPFHk+oHwVE/TRqd1kE
+p+f2b3BofRMG5TYPZ3POF+wljrP0VCAB3Joy32716VLrJcdEpE9svVo67AsOA7qJ
+zrhsRUznXghVXdNqO8IIPLGJUw4bxcPAp+A7ragf76TopZOS8qCnFurysUkZBdg9
+lRhvF5HpRONJI/IaetrSuG+s8422KdWkXnUv5hZTbwNirg0p7lUx6Fl3PQNZ+jiP
+QAATrSAfduZ3vnasTq7VBpuDOamFug4br9i3JRqrQCF40dMCSJLkWW7kI4n8Jr9N
+/9jWkXiNtUCCm67UCFjZUwdbsbxr544q1Jvt+yUx1qWmXu6QaedyPJsqtS1rculH
+20a9euRUJ8Q72L1faOROJepjPdnARunasbzI2G+PGczuJp0jRMgKs6u/OJZh1RVr
+FVmK18wTk/W+dX0ojoLpSddbCGDoSQBYjQCz4D6LatdCsW/+4vvP6k6pht6mEQyk
+7yQNzmrLOdgZDXxU3NfhjBDCa7oVqEaL5DKTL5t1NQrutfPbv2ixKLRqETYA5qGd
+cxJJmQzzafyh7m5wET2OeaCHSILzCW4k9TdXqEKAfst/mK3fDMWSQZeO8Ld9wDqA
+kgh1NM62ncukyBwJI5ImEKmlhNe3rmn0oNkRqgQD2DLAgcfOESqZ/61j2GGPnsjg
+xkaeKfVJHA==
+=XTEI
 -----END PGP PUBLIC KEY BLOCK-----
diff --git a/swaylock.spec b/swaylock.spec
index 1ddcf3d..97b7d60 100644
--- a/swaylock.spec
+++ b/swaylock.spec
@@ -11,7 +11,7 @@ Source1:    %{url}/releases/download/v%{version}/%{name}-%{version}.tar.gz.sig
 Source2:    gpgkey-E88F5E48.asc
 
 BuildRequires:  gcc
-BuildRequires:  gnupg2
+BuildRequires:  openpgpverify
 BuildRequires:  meson >= 0.59.0
 BuildRequires:  pam-devel
 BuildRequires:  pkgconfig(cairo)
@@ -26,7 +26,7 @@ BuildRequires:  scdoc
 swaylock is a screen locking utility for Wayland compositors.
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %autosetup
 
 %build
-- 
2.56.0

